lib/brains/models.py

```diff
@@ -24,12 +29,41 @@ def get_uri(self):
         return reverse('braintree:mozilla:buyer', kwargs={'pk': self.pk})
 
 
+@receiver(Buyer.close_signal, sender=Buyer)
+def close(signal, *args, **kw):
+    buyer = kw['buyer']
+    try:
+        braintree_buyer = BraintreeBuyer.objects.get(buyer=buyer)
+    except BraintreeBuyer.DoesNotExist:
+        # The buyer might not have used braintree. If that's the
+        # case continue.
+        log.info('No braintree buyer found for buyer: {}'
+                 .format(buyer.pk))
+        return
+
+    for paymethod in braintree_buyer.paymethods.all():
+
+        # Find and clear out all subscriptions.
+        for subscription in paymethod.subscriptions.filter(active=True):
+            subscription.braintree_cancel()
```

Comment:
Currently it call the braintree API, which will then trigger the webhooks, which will then come back to payment-service, hit solitude and generate an email. So yes, they'll get an email. Unless you've got a ton of accounts, I still kind of like that idea of getting the confirmation by email.